test/python/transpiler/test_high_level_synthesis.py

```diff
@@ -785,6 +815,842 @@ def test_reordered_modifiers(self):
         self.assertEqual(Operator(transpiled1), Operator(transpiled2))
         self.assertEqual(Operator(qc1), Operator(transpiled1))
 
+    def test_definition_with_annotations(self):
```

Comment:
Are there any tests with `top_level_only=True`? If not we should add some.

Reply:
Solved by removing `top_level_only`. :)